components/41-organisms/collection/_collection.scss

```diff
@@ -7,13 +7,13 @@ ul.grid-5 {
   display: flex;
   flex-wrap: wrap;
   width: calc(100% + #{gutter()});
-  margin: 0 -#{gutter()/2} -1.2rem;
+  margin: 0 -#{gutter()*.5} -1.2rem;
 
   > li {
     flex-basis: 100%;
     flex-shrink: 1;
     margin-bottom: 1.2rem;
-    padding: 0 gutter()/2;
+    padding: 0 gutter()*.5;
```

Comment:
Add space between * and .5. Makes it clearer that it is a calculation instead of a wildcard.